mkosi.extra/usr/lib/systemd/mkosi-check-and-shutdown.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -eux
+# SPDX-License-Identifier: LGPL-2.1-or-later
```

Comment:
```suggestion
#!/bin/bash
# SPDX-License-Identifier: LGPL-2.1-or-later
set -ex
```
this way we don't get noise from bash config files being read. Dropped the `-u`, since there are no variables, but it also doesn't hurt if it's there.